AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLRequest/GraphQLRequest+Model.swift

```diff
@@ -149,6 +149,12 @@ extension GraphQLRequest: ModelGraphQLRequestFactory {
         return mutation(of: model, modelSchema: modelSchema, where: predicate, type: .delete)
     }
 
+    public static func mutation<M: Model>(of model: M,
+                                          where predicate: QueryPredicate? = nil,
+                                          type: GraphQLMutationType) -> GraphQLRequest<M> {
+        return mutation(of: model, modelSchema: model.schema, where: predicate, type: type)
```

Comment:
nit: remove redundant `return`
```suggestion
        mutation(of: model, modelSchema: model.schema, where: predicate, type: type)
```